jwst/datamodels/schemas/keyword_band.schema.yaml

```diff
@@ -9,5 +9,6 @@ properties:
           band:
             title: MRS wavelength band
             type: string
-            enum: [SHORT, MEDIUM, LONG, N/A, ANY, MULTIPLE]
+            enum: [SHORT, MEDIUM, LONG, SHORT-MEDIUM, SHORT-LONG, MEDIUM-SHORT, MEDIUM-LONG,
+	    LONG-SHORT, LONG-MEDIUM, N/A, ANY, MULTIPLE]
```

Comment:
Continuation lines must be intended to match the start of the list in the preceding line. This is what's causing the Travis errors (I believe).